controllers/keystoneapi_controller.go

```diff
@@ -794,11 +794,10 @@ func (r *KeystoneAPIReconciler) ensureFernetKeys(
 			},
 		}
 		err := oko_secret.EnsureSecrets(ctx, helper, instance, tmpl, envVars)
+
 		if err != nil {
-			return nil
+			return fmt.Errorf("OpenStack secret %s not found", secretName)
```

Comment:
Why override the error coming from lib-common? just 'return err' would be better I think.

Reply:
totally agree. Updated the change accordingly.